llm/requirements.txt

```diff
@@ -0,0 +1,2 @@
+python-dotenv
+wandb
```

Comment:
Do you need to pin versions of these in case they change drastically in the future?

Reply:
Done.